blocks/v2-testimonial/v2-testimonial.css

```diff
@@ -0,0 +1,174 @@
+/* stylelint-disable selector-class-pattern */
+
+.v2-testimonial {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.v2-testimonial__blockquote-column {
+  background-color: #2b2b2b;
+  padding: 48px 16px;
+  color: var(--c-primary-white);
+  font-family: var(--ff-body);
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 150%;
+  letter-spacing: 0.3px;
+  padding-right: calc(100vw - 510px);
+}
+
+.v2-testimonial__blockquote-column strong {
+  font-family: var(--ff-body-bold);
+}
+
+.v2-testimonial__heading {
+  color: var(--c-primary-white);
+  font-family: var(--ff-headline-medium);
+  font-size: 32px;
+  font-style: normal;
+  line-height: 115%;
+  letter-spacing: -0.64px;
+  margin: 24px 0;
+}
+
+.v2-testimonial__blockquote {
+  font-family: var(--ff-testimonial);
+  font-size: 18px;
+  font-style: normal;
+  line-height: 120%;
+  letter-spacing: -0.36px;
+  margin: 40px 0;
+  text-indent: 0;
+}
+
+.v2-testimonial__image {
+  width: 100%;
+  display: flex;
+  aspect-ratio: 16/9;
+  object-fit: cover;
+  object-position: center;
+}
+
+.v2-testimonial__video-section {
+  padding: 20px 16px;
+  background-color: var(--c-primary-gold);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 1;
+  gap: 8px;
+}
+
+.v2-testimonial__author {
+  display: inline-flex;
+  gap: 16px;
+  align-items: center;
+  margin: 0;
+  font-size: 13px;
+}
+
+.v2-testimonial__video-link-wrapper {
+  margin: 0;
+  display: flex;
+}
+
+.v2-testimonial .v2-testimonial__video-section .v2-testimonial__video-link {
+  margin: 0;
+  padding: 4px 0;
+  color: var(--c-primary-black);
+  font-family: var(--ff-body);
+  font-size: 14px;
+  font-style: normal;
+  line-height: 18px;
+  letter-spacing: 1.12px;
+  background: transparent;
+  border: 0;
+  display: inline-flex;
+  gap: 8px;
+  align-items: center;
+}
+
+.v2-testimonial .v2-testimonial__video-section .v2-testimonial__video-link:hover {
+  text-decoration: underline;
+}
+
+.v2-testimonial__video-link::before {
+  content: '';
+  height: 32px;
+  width: 1px;
+  background: var(--c-accent-copper);
+}
+
+.v2-testimonial__video-link .icon {
+  height: 24px;
+}
+
+.v2-testimonial__video-link svg {
+  height: 24px;
+  width: 24px;
+}
+
+.v2-testimonial__text-row {
```

Comment:
Is the order necessary? If we are positioning the elements absolutely.

Reply:
On mobile, there is no absolute positioning. The order should always be testimonial text at the top and image at the bottom - it doesn't depend on the doc's content.